general/toc.yml

```diff
@@ -25,14 +25,6 @@
   - uid: network-letsencrypt
   - uid: network-dlna
 
-- uid: contrib-index
-  items:
-  - uid: contrib-branding
-  - uid: contrib-development
-  - uid: contrib-issues
-  - uid: contrib-release-procedure
-  - uid: contrib-source-tree
```

Comment:
Why move this outside the main table of contents?

Reply:
Personally, I found the contributing documentation very difficult to find. I think it's better to have it right in the header nav in it's own section. It also doesn't really make sense to have it under "Quick Start". This way, the quick start section is dedicated to content for users, and the "Contributing" section is dedicated to content only for developers.

Reply:
This comparison makes the difference more clear. The contribution instructions have much more visibility and specifically it's much easier to get the development documentation (not hidden behind a collapsed menu)
![image](https://user-images.githubusercontent.com/2626103/76166913-a61f9b80-6162-11ea-9514-361753c1d055.png)
![image](https://user-images.githubusercontent.com/2626103/76166928-c8b1b480-6162-11ea-8967-189c37bfef14.png)

Reply:
I also considered moving the "Plugin API" menu item down into the contributing section to reduce the number of items in the main navbar. Can make that change as well if you think it's a good idea.

Reply:
I think that's definitely a good idea.

Reply:
Alright, I've moved `Plugin API` to a subheading under `Contributing`